src/components/Range/index.tsx

```diff
@@ -361,33 +406,37 @@ const Range = ({
   )
 
   const onMouseUp = useCallback(() => {
-    container.current.removeEventListener('mousemove', onMouseMove)
-    grabCursor()
+    if (container.current) {
+      container.current.removeEventListener('mousemove', onMouseMove)
+      grabCursor(undefined)
```

Comment:
This would mean grabCursor has an optional first parameter, there might be an issue with the setState which should be able to accept undefined

Reply:
I don't understand your comment. Here it sets cursor to undefined. And we cannot leave it empty otherwise TS complains.

It doesn't shock me to specify the value becomes undefined ?